LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) [year] [fullname]
+Copyright (c) [2019] [Code Carbon]
```

Comment:
```suggestion
Copyright (c) 2019 Mila, BCG Gamma, CometML, Haverford College
Copyright (c) 2023 Code Carbon NGO
```

Reply:
LGTM! We need explicit consent from Victor, and maybe precise that the previous copyright was 2019-2023.